typescript.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
         "react/prop-types": "off",
         "@typescript-eslint/array-type": ["error", { default: "generic", readonly: "generic" }],
         "@typescript-eslint/ban-types": "error",
+        "@typescript-eslint/ban-ts-ignore": "error",
```

Comment:
please move it up one line - it is sorted alphabetically (at least it should be)

Reply:
https://github.com/pmedianetwork/eslint-config/pull/58/commits/30c5e3d8e58fcfa204dc503916e3572d4193e3f9